### ISSUE TITLE
Not showing version at MacOS

### ISSUE THREAD

**Issue by jfagoagas** [rhyeal/aws-rotate-iam-keys#61]
Hi!

`aws-rotate-iam-keys` is not showing version number when you input `--version` in MacOS.

I think `<%VERSION%>` from `echo 'version <%VERSION%>'` at https://github.com/rhyeal/aws-rotate-iam-keys/blob/master/src/bin/aws-rotate-iam-keys#L60 can't be interpreted.

```
$ /tmp aws-rotate-iam-keys --version
AWS Rotate IAM Keys (c) 2018+ Adam Link.
Licensed under the GNU General Public License.
Thanks to all the contributors!
version <%VERSION%>
$ /tmp sw_vers
ProductName:	macOS
ProductVersion:	12.0.1
BuildVersion:	21A559
```

Some debug information with `bash -x aws-rotate-iam-keys --version`:
```
+ test -t 1
+ set -eu -o errexit -o pipefail -o noclobber -o nounset
+ IFS='
	'
+ getopt --test
+ [[ 0 -ne 4 ]]
+ which brew
++ brew --prefix
+ test -d /usr/local/opt/gnu-getopt/bin
++ brew --prefix
+ PATH=/usr/local/opt/gnu-getopt/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin
++ getopt --options=hvfp: --longoptions=force,profile:,profiles:,version,help --name aws-rotate-iam-keys -- --version
+ PARSED=' --version --'
+ [[ 0 -ne 0 ]]
+ eval set -- ' --version --'
++ set -- --version --
+ PROFILES=
+ FORCE=
+ true
+ case "$1" in
+ echo 'AWS Rotate IAM Keys (c) 2018+ Adam Link.'
AWS Rotate IAM Keys (c) 2018+ Adam Link.
+ echo 'Licensed under the GNU General Public License.'
Licensed under the GNU General Public License.
+ echo 'Thanks to all the contributors!'
Thanks to all the contributors!
+ echo 'version <%VERSION%>'
version <%VERSION%>
+ exit 0
```

**Comment by mmrwoods** [comment]
Fixed by commit 53d15acca2a43d235dfb34dfbd26932fb3b0134d

**Comment by jfagoagas** [comment]
Thanks @mmrwoods !!
